### DISCUSSION TITLE
Discharging rate is much higher than anticipated regardless of Power sliders. (Issue start last month or so)

### DISCUSSION THREAD

**Discussion by NomadicMeow** [seerge/g-helper#3083] [Q&A]
I have an issue that no matter what my SPL, sPPT and fPPT is set to, it seems to not care. I currently have it on Silent mode with 7W, 7W and 7W for SPL, sPPT and fPPT, respectively. The discharging rate still goes up and stays well above my limits for a long time with no regards what so ever to the limits set. 
![Screenshot 2024-09-09 184132](https://github.com/user-attachments/assets/3a91ba82-21ef-4aa3-853c-18a5bbf7d9ef)
HWMonitor shows the PPT set to 54W regardless of what I do and Universal x86 Tuning utility seem to be the only one that can change it. BUT, even with that my discharge rates are still higher than before and seem to persist.

![Screenshot 2024-09-09 112302](https://github.com/user-attachments/assets/e27717e4-496e-47fc-8feb-1d952a0ece58)
with G-Helper
![Screenshot 2024-09-09 112702](https://github.com/user-attachments/assets/38568b04-6f01-4dd6-9f9a-97ffd14ecc56)
with Universal x86 Tuning Utility

I understand that Wi-Fi card, RAM, SSD, Display Brightness, Bluetooth, Backlight can contribute but again, this was NOT an issue before. I have tried turning off/lowering down everything but it seem to not help at all!

**Comment by seerge** [comment]
@NomadicMeow hello.

Discharge rate includes not only CPU / APU. It also includes display backlight, SSD,  power that motherboard consumes and some random things as well. So it can be higher than the limits you have set.

Also discharge rate coming from windows (G-Helper takes it there) usually lags behind actual instant consumption. So it may settle down to a lower amount with some time.

As for the limits - HWmonitor shows a bunch of different things. What is important **is actual current power consumption by CPU** - `1.22W` and APU (CPU + iGPU) - `3.6W`. So it's all good and within limits that you have set :) Also it seem to show all 3 limits correctly `7/10/15W` correspondingly (even tho it calls them incorrectly). I don't know what it means under PPT, but it's not a CPU limit for sure.

---

Try to reboot your laptop and set **Battery Saver** mode in windows, it usually helps a lot to stop random tasks windows likes to run in background.

**Reply by NomadicMeow** [reply]
Thank you seerge for replying back. I do understand all of that and I tried everything including what you stated, BUT this was NOT an issue before (even WITHOUT doing the measures to try to minimize the power consumption). My last resort is to reinstall Windows 11 from scratch using a bootable drive, because it is really frustrating, especially while having a battery with 72% health remaining. 

I have to also add, many many thank yous for making G-Helper.

**Reply by seerge** [reply]
@NomadicMeow it could be that discharge rate is just a bit inaccurate (i assume you see same discharge in hwinfo / g-helper / any other monitoring tool). 

- Try to recalibrate battery by discharging to low level and charging back to 100% 
- Try [hard reset](https://github.com/seerge/g-helper/wiki/Troubleshooting#hard-reset--embedded-controller-ec-reset) to see if it helps as well
